src/js/components/Navigation/FooterCandidateList.jsx

```diff
@@ -19,9 +21,33 @@ export default function FooterCandidateList () {
         stateNamePhrase = `${stateName}-candidates`;
         stateNamePhraseLowerCase = stateNamePhrase.replace(/\s+/g, '-').toLowerCase();
         // console.log('tempStateCode:', tempStateCode, ', stateAlreadySelected:', stateAlreadySelected);
+        const linkTo = `/${stateNamePhraseLowerCase}/cs/`;
+
+        function handleClick() {
```

Comment:
Hi @kjbyun0, including the handleClick() function in the loop makes me nervous. Could you move handleClick before the return block, and instead pass "currentPathName" and "linkTo" into handleClick when it is called?